### ISSUE TITLE
Zoom does not work

### ISSUE THREAD

**Issue by collimarco** [lyfeyaj/swipe#114]
You cannot zoom the slides and this creates accessibility issues. Users should always be able to use their browser zoom (i.e. pintch-to zoom).

**Comment by loup-brun** [comment]
there is the `disableScroll` option, which prevents other touch events such as scrolling.

can you confirm you see that issue with the `disableScroll` option set to `false`? (it is `false` by default)

on my iPhone I can pinch-zoom without any problem.
what device are you using?
